bitnami/redis-cluster/values.yaml

```diff
@@ -20,6 +20,11 @@ global:
   redis:
     password: ""
 
+## @param emptyDir.sizeLimit Size limits on emptyDir
+##
+emptyDir:
+  sizeLimit: 512Mi
```

Comment:
LGTM but I think that the value should be empty to customize it depending on personal preferences.